shroudstone/renamer.py

```diff
@@ -76,7 +76,7 @@ def migrate():
             last_run_version_file.read_text(encoding="utf-8")
         )
     else:
-        last_run_version = version.parse("0.1.0a29")
+        last_run_version = version.parse(__version__)
```

Comment:
This will mean migrations won't run at all the first time a user launches the app, which isn't great. 
Right now this has no impact because the cache files are no longer used anyway, but could break things in the future.
I'll switch this back.